Resources/Prototypes/_NF/Shipyard/garden.yml

```diff
@@ -0,0 +1,27 @@
+- type: vessel
+  id: Garden
+  name: HS Garden
+  description: A small botany vessel dedicated to horiticultural experimentation.
+  price: 29750
```

Comment:
it was only worth around 16000 when I spawned it in. Im gunna change this to be a tad cheaper, maybe make it just barely purcahaseable for a new character.